models/sales/sales.yml

```diff
@@ -19,6 +19,9 @@ models:
         description: The ID of the deal
         tests:
           - unique
```

Comment:
Same comment as below

Reply:
Just bumping this as I sent the review out too early and you see my above comment.

Reply:
addressed